admin_guide/scheduling/scheduler.adoc

```diff
@@ -446,44 +444,40 @@ see xref:../../admin_guide/scheduling/pod_placement.adoc#controlling-pod-placeme
 Multiple-level labels are also supported. Users can also specify all pods for a service to
 be scheduled on nodes within the same region and within the same zone (under the region).
 
-**_LabelsPresence_** checks whether a particular node has a certain label
-defined or not, regardless of its value. Matching by label can be useful, for
-example, where nodes have their physical location or status defined by labels.
+The `labelsPresence` parameter checks whether a particular node has a specific label. The labels create node _groups_ that the
+`LabelPreference` priority uses. Matching by label can be useful, for example, where nodes have their physical location or status defined by labels.
 
 [source,json]
 ----
 "predicates":[
       {
          "name":"<name>", <1>
-         "weight" : "1" <2>
          "argument":{
             "labelsPresence":{
                "labels":[
-                  "<label>" <3>
-                presence: true/false <4>
+                  "<label>" <2>
+                presence: "true" <3>
```

Comment:
here, the structure should be:

```
"predicates":[
      {
         "name":"<name>", (1)
         "argument":{
            "labelsPresence":{
               "labels":[
                  "<label>" (2)
                ],
                "presence": true (3)
            }
         }
      }
   ],
```

Reply:
Made this change in a follow-up PR: https://github.com/openshift/openshift-docs/pull/11335